llvm/test/Transforms/LoopVectorize/vplan-printing.ll

```diff
@@ -599,7 +602,7 @@ define void @print_expand_scev(i64 %y, ptr %ptr) {
 ; CHECK-NEXT: Successor(s): middle.block
 ; CHECK-EMPTY:
 ; CHECK-NEXT: middle.block:
-; CHECK-NEXT:   EMIT vp<[[CMP:%.+]]> = icmp eq vp<[[TC]]>, vp<[[VEC_TC]]>
+; CHECK-NEXT:   EMIT vp<[[CMP:%.+]]> = icmp eq vp<[[TC]]>, vp<[[VTC]]>
```

Comment:
VEC_TC >> VTC?

Reply:
The test originally use d`VEC_TC` defined by a VPlan of an earlier function, while it should match `VTC` from the match for this function, which was OK so far. Pushed 34034381b7d54da864f8794f578d9c501d6d4f3b to consistently use VTC across the file, removing the diff here